### ISSUE TITLE
Add theme suggestions for Views

### ISSUE THREAD

**Issue by lramarojaona** [openfed/kiso#103]
Adding template suggestions for Views could be helpful:

```php
/**
 * Implements hook_theme_suggestions_HOOK_alter().
 */
function HOOK_theme_suggestions_views_view_alter(array &$suggestions, array $variables) {
  $suggestions[] = 'views_view__' . $variables['view']->id();
}
```

**Comment by lramarojaona** [comment]
This snippet, might need improvement/fix, as I'm getting multiple identical suggestions:
![2023-08-31 16_47_54-Burn-out _ FEDRIS — Mozilla Firefox](https://github.com/openfed/kiso/assets/5821299/fc09d57c-f8d4-45ac-ab74-869b53004896)

**Comment by jclemmenb4y** [comment]
Do we really need this? Template suggestion for Views are available in core, see https://www.drupal.org/docs/develop/theming-drupal/twig-in-drupal/twig-template-naming-conventions#s-views

The issue with the suggestions for Views specifically is that they don't show up when debug is enabled, but the naming convention is pretty clear and simple.

What do you think @rutiolma ?

**Comment by rutiolma** [comment]
Having a debug to know the views template to use is very useful and I often miss it but I also agree that the documentation is pretty clear and simple.

The proposed hook, besides the repeated debug info, is a bit limited since it works only for the view template but not for the displays or fields. Also, the repeated template suggestion, is due to a core issue so we cannot do much but patch it.
I see that there are several core issues trying to tackle this views template issue and https://www.drupal.org/project/drupal/issues/2118743 is pretty active and is the one that better fixes the issue but, until there's a stable patch, I don't think we should be adding patches to core just for debug.

So, in short, I agree with the necessity to have template suggestions but I don't think we have a good way to fix it.
I would suggest postponing this issue until there's a core improvement.
